packages/app-store/zoho-bigin/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "Zoho Bigin",
+  "slug": "zoho-bigin",
+  "type": "zoho-bigin_other_calendar",
+  "imageSrc": "icon.svg",
```

Comment:
```suggestion
```
logo is enough. imageSrc is deprecated now. App Store CLI shouldn't add it for a new app.